### DISCUSSION TITLE
Is it possible to findUnique(), but case insensitive?

### DISCUSSION THREAD

**Discussion by lumenwrites** [prisma/prisma#14449] [Q&A] [answered]
This works:
```javascript
  const coupon = await prisma.coupon.findMany({
    where: {
      code: {
        equals: couponCode,
        mode: 'insensitive'
      }
    }
  })
```

But I'd like to be able to use `findUnique()` instead:
```javascript
  const coupon = await prisma.coupon.findUnique({
    where: {
      code: {
        equals: couponCode,
        mode: 'insensitive'
      }
    }
  })
```

Do you know if that's possible?

**Comment by nurul3101** [comment, marked as answer]
Hey, @lumenwrites 👋 
`mode` parameter would work in `findMany` or `findFirst` queries but isn't supported in `findUnique`.

You would need to pass the exact text based on which you want to filter in `findUnique`, Also only the fields which are marked with `@unique` attribute can be used in the where clause.

**Reply by lumenwrites** [reply]
Thank you!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
